lib/src/Recorder.cpp

```diff
@@ -97,6 +97,58 @@ bool Recorder::setRecordingOutputDirectory(
     return true;
 }
 
+bool Recorder::setRecordingFilename(
+    _In_ const sai_attribute_t &attr)
+{
+    SWSS_LOG_ENTER();
+
+    if (attr.value.s8list.count == 0)
+    {
+        m_recordingFileName = "sairedis.rec";
```

Comment:
this name is also used in constructor, can you make a define on top of file and reuse define here ?

Reply:
address the commit in the latest commit